### PR TITLE
Mhs/cumulus 2224/add extended metadata type to reconciliation report

### PR DESCRIPTION
**Pull request by flamingbear**
**Summary:** Adds "EXTENDED METADATA" type to granule files' comparisons

Addresses [CUMULUS-2224: Not all files are being found in CMR when they exist in cumulus during reconciliation reporting](https://bugs.earthdata.nasa.gov/browse/CUMULUS-2224)

## Changes

* Adds test and code to include the relatedUrls Type 'EXTENDED METADATA' 

## PR Checklist

- [ x] Update CHANGELOG
- [ x] Unit tests
- [ ] Adhoc testing
- [ ] Integration tests

### CODE REVIEW

File: packages/api/tests/lambdas/test-create-reconciliation-report.js [hunk 1]
```diff
@@ -1434,7 +1434,12 @@ test.serial('reconciliationReportForGranuleFiles reports discrepancy of granule
   },
   {
     URL: `${process.env.DISTRIBUTION_ENDPOINT}testbucket-protected-2/MOD09GQ___006/MOD/MOD09GQ.A4675287.SWPE5_.006.7310007729190.cmr.xml`,
-    Type: 'GET DATA',
+    Type: 'EXTENDED METADATA',
+    Description: 'File to download',
+  },
+  {
+    URL: 'http://example.com/thisFileIsIgnoredBecauseOfTheRelatedUrlType.exe',
+    Type: 'DOWNLOAD SOFTWARE',
```
Comment (jennyhliu) [inline packages/api/tests/lambdas/test-create-reconciliation-report.js:1442]: The variable is `matchingFilesInCmr`, need another variable, maybe `ignoredFilesInCmr`?

Comment (flamingbear) [reply packages/api/tests/lambdas/test-create-reconciliation-report.js]: good point. ed2dd430a

**Review by jennyhliu — APPROVED** [review]
LGTM

**Comment by flamingbear** [comment]
waiting for @laurenfrederick approval to merge. (we might not want to update this)

**Pull request merged**